aten/src/ATen/cpu/vec/vec512/vec512.h

```diff
@@ -97,7 +107,7 @@ inline mask_gather(const Vectorized<double>& src, const double* base_addr,
 template<int64_t scale = 1>
 std::enable_if_t<scale == 1 || scale == 2 || scale == 4 || scale == 8, Vectorized<float>>
 inline mask_gather(const Vectorized<float>& src, const float* base_addr,
-                   const Vectorized<int32_t>& vindex, const Vectorized<float>& mask) {
+                   const Vectorized<int32_t>& vindex, Vectorized<float>& mask) {
```

Comment:
Why is this change necessary?

Reply:
because generic template signature does not have `const` for mask. WIth the `const` here, this method was never called.